src/index.js

```diff
@@ -1,3 +1,20 @@
-import { getNearestShops } from './app.js';
+import { getNearestShops } from "./app.js";
+import { ERROR_TOKEN } from "./utils.js";
 
-getNearestShops();
+// Get command arguments
+const args = process.argv.slice(2);
+const position = { x: args[1], y: args[2] };
+
+// Get nearest coffee shops
+const shops = await getNearestShops(position);
+
+// Notify and exit if error occurred
+if (shops[0] === ERROR_TOKEN) {
+  console.log("Operation failed. Error Code: " + shops[1]);
+  process.exit();
+}
+
+// Log the 3 closest coffee shops
+shops.forEach(function (coffeeShop) {
+  console.log(coffeeShop.name + ", " + coffeeShop.delta);
+});
```

Comment:
Regarding single responsibility principle. What are the possible reasons why you would have to make changes in this file? I see at least 2 reasons.